### PR TITLE
fixed component structure to adaptors/{framework} for components

### PR DESCRIPTION
**Pull request by terencejeong**
Folder restructure so now we follow the pattern 

|-- components (as is now)
|-- adapters {
      |-- react
      |-- vue
      |-- angular
}

Please make sure to pull the branch and test making a new React component with plop. 

The key changes with this PR are related to the plopfile.js 
- Changed the REACT_PATH and VUE_PATH
- Also changed the checkComponent function path to 'adaptors/react/etc'

Since I am new to monorepos please check the package.json file and lerna.json > workspaces and if that makes sense.

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-import { configure, addDecorator, addParameters } from '@storybook/react';
```
Comment (tilanka) [inline .storybook/config.js]: are we doing `space-in-brackets` @jennasalau ?

Comment (terencejeong) [reply .storybook/config.js]: god damn it, no i thought I fixed that. Will update PR, I also spelt adapters wrong...

Comment (jennasalau) [reply .storybook/config.js]: lol this made me laugh..

Yeah no we are not Til, same with arrays incase you see it

**Comment by terencejeong** [comment]
So rebased branch off rhythm-ui-mk-2 hence reason for force push, still need to investigate why these checks are failing.

**Comment by jennasalau** [comment]
Hey @terencejeong can we please fix the last conflict tomorrow :)

**Comment by terencejeong** [comment]
Sure! Sorry about the delay

**Pull request merged**